use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,358 @@
+#Introduction
+
+Intel\* QuickAssist Technology (Intel\* QAT) provides hardware acceleration of security (cryptography) and compression. These instructions cover the steps for Clear Linux\* but can be adapted to any Linux\* distribution. Your distribution may already have the Intel QAT drivers, but it is likely they are out of date and do not contain the necessary user space components.  These instructions guide the user on how to download the kernel sources, compile kernel driver modules against those sources, and load them onto the host as well as preparing a specially built Kata Containers® kernel and custom Kata Containers rootfs.  
```

Comment:
You can probably drop the `\*`'s, as we don't have a footer for them to reference, and the kata/OSF project docs don't use the idiom.

Reply:
The technical content services team suggested adding them and at the very bottom is the other brands and trademarks reference.

Reply:
For Intel published docs, sure. But, we asked recently about this for docs on the Kata repos with the OSF, and they said they don't do it, so drop them. I'd drop them here.